lib/engine/game/g_1817/step/share_buying_with_shorts.rb

```diff
@@ -9,6 +9,7 @@ module ShareBuyingWithShorts
         # such as in 1889 for exchanging Dougo
         #
         def can_gain?(entity, bundle, exchange: false)
+          return if bundle.owner&.player?
```

Comment:
Should this e in the other order like where it changed in 1822?

Reply:
Yeah